Modules/faulthandler.c

```diff
@@ -1107,8 +1107,8 @@ faulthandler_fatal_error_c_thread(PyObject *self, PyObject *args)
     Py_RETURN_NONE;
 }
 
-static PyObject* _Py_NO_SANITIZE_UNDEFINED
-faulthandler_sigfpe(PyObject *self, PyObject *args)
+static PyObject * _Py_NO_SANITIZE_UNDEFINED
```

Comment:
As the faulthandler module maintainer, I prefer `PyObject*` :-)

```suggestion
static PyObject* _Py_NO_SANITIZE_UNDEFINED
```